c/examples/tcp_2sigma_example/tcp_2sigma_example.cc

```diff
@@ -0,0 +1,267 @@
+/*
+ * Example demonstrating how to parse SBP data from a TCP port and compute the
+ * ellipse representing the 2 sigma confidence level.
+ *
+ * Requires libsbp (https://github.com/swift-nav/libsbp) and Eigen 3 (https://eigen.tuxfamily.org/).
+ * Eigen 3 is provided by the libeigen3-dev package on Debian-like Linux distributions.
+ *
+ * Compile using:
+ * g++ tcp_2sigma_example.cc -I ${LIBSBP}/c/include/ -L ${LIBSBP}/c/build/src/ -lsbp
+ *
+ * If compiled with -DSHOW_PLOT then the ellipses will be graphically displayed.
+ * This functionality requires OpenCV3 (https://opencv.org/) which is provided
+ * by the libopencv-dev package on Debian-like Linux distributions.
+ *
+ * Compile using:
+ * g++ tcp_2sigma_example.cc -DSHOW_PLOT -I ${LIBSBP}/c/include/ -L ${LIBSBP}/c/build/src/ -lsbp -lopencv_core -lopencv_imgproc -lopencv_highgui
+ *
+ * Originally adapted from https://github.com/swift-nav/libsbp/blob/master/c/example/tcp_example/tcp_example.c
+ */
+
+#ifndef SHOW_PLOT
+#define SHOW_PLOT 0
```

Comment:
@dgburr is it worth installing opencv in CI so we can test compilation of the plotting code?

Reply:
Sorry, but I didn't see this comment before I merged (didn't hit refresh).  Please LMK if you think this is important, in which case I will open a new PR.

Reply:
@dgburr my concern would be around bitrot, if we can compile it easily it'll be easier to keep the code healthy in the long term